Samsung/A31/res/xml/power_profile.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<device name="Android">
+    <item name="ambient.on">0.1</item>
+    <item name="screen.on">0.1</item>
```

Comment:
A screen that consumes 0.1mA at max  brightness? awesome ! Sounds physically impossible but well...